src/components/Api.js

```diff
@@ -0,0 +1,83 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  _handleResponse(res) {
+    if (res.ok) {
+      return res.json();
+    } else {
+      return Promise.reject("Error ${res.status}");
```

Comment:
`"Error ${res.status}"` 


You need to use `backticks` for string templates to insert some data into the string


```js

      return Promise.reject(`Error ${res.status}`);

```